src/api.php

```diff
@@ -600,17 +644,20 @@
             $zData['_position_mRNA'] = explode(';', $zData['_position_mRNA']);
         }
 
+        // Adding the genomic variants and positions for each build individually.
+        $aGenomicVariants = array();
+        $aGenomicPositions = array();
+        foreach ($aActiveGBs as $sGBID => $sGBSuffix) {
+            $aGenomicVariants[$sGBID] = 'chr' . $sChromosome . ':' . $zData['VariantOnGenome/DNA' . (!$sGBSuffix? '' : '/' . $sGBSuffix)];
+            $aGenomicPositions[$sGBID] = $zData['position_genomic' . (!$sGBSuffix? '' : '_' . $sGBSuffix)];
+        }
+
         $aReturn = array(
             'symbol' => $sSymbol,
             'id' => $zData['id'],
             'position_mRNA' => $zData['_position_mRNA'],
-            'position_genomic' => array(
-                $_CONF['refseq_build'] => $zData['position_genomic'],
-            ),
-            'variant_genomic' => array(
-                $_CONF['refseq_build'] => 'chr' . $sChromosome . ':' .
-                    $zData['VariantOnGenome/DNA'],
-            ),
+            'position_genomic' => $aGenomicPositions,
+            'variant_genomic' => $aGenomicVariants,
```

Comment:
Note that in the original code, `position_genomic` and `variant_genomic` were only returned as arrays if we asked for JSON output. Which genome build is now selected when we don't ask for JSON output? Is that still always the "correct" one? (I don't think so)